ecommerce/src/test/kotlin/io/hhplus/ecommerce/support/KotestIntegrationTestBase.kt

```
@@ -37,6 +37,26 @@ abstract class KotestIntegrationTestBase(body: DescribeSpec.() -> Unit = {}) : D
+                val categoryExists = jdbcTemplate.queryForObject(
+                    "SELECT COUNT(*) FROM categories WHERE id = 1",
+                    Int::class.java
+                ) ?: 0
+
+                if (categoryExists == 0) {
+                    jdbcTemplate.execute("""
+                        INSERT INTO categories (id, name, description, display_order, created_at, updated_at)
+                        VALUES (1, '기본 카테고리', '테스트용 기본 카테고리', 0, NOW(), NOW())
+                    """.trimIndent())
+                }
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)